frontend/playwright.config.ts

```diff
@@ -50,6 +50,9 @@ const config: PlaywrightTestConfig = {
       trace: 'on-first-retry',
 
       viewport: { width: 1400, height: 1000 },
+      launchOptions: {
+         slowMo: 1000,
```

Comment:
Interesting. How much does this slow down the test?

Reply:
Without this the tests fail, but if were to just run the tests on Chromium with 4 parallel workers. Then it slows it down by about 20 seconds.

Reply:
<img width="742" alt="Test Failure@2x" src="https://user-images.githubusercontent.com/22064420/208158648-4e3bfbc3-14f6-43a0-b127-92ee013fad68.png">